security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -12,6 +12,34 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-47"]
+== OpenShift Compliance Operator 0.1.47
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.47:
+
+* link:https://access.redhat.com/errata/RHBA-2022:0014[RHBA-2022:0014 OpenShift Compliance Operator Bug Fix and Enhancement Update]
+
+[id="compliance-operator-0-1-47-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The Compliance Operator now supports the following compliance benchmarks for the Payment Card Industry (PCI) Data Security Standard:
++
+** ocp4-pci-dss
+** ocp4-pci-dss-node
+
+* Additional rules and remediations for FedRAMP moderate are added to the OCP4-moderate, OCP4-moderate-node, and rhcos4-moderate profiles.
+
+* Remediations for KubeletConfig are now available in node-level profiles.
+
+[id="openshift-compliance-operator-0-1-47-bug-fixes"]
+=== Bug fixes
+
+* Previously, if your cluster was running OCP 4.6 or earlier, remediations for USBGuard-related rules would fail for the moderate profile. This is because the remediations  created by the Compliance Operator  were based on an older version of USBGuard that did not support drop-in directories. Now, invalid remediations for USBGuard-related rules are not created for clusters running OCP 4.6. If your cluster is using OCP 4.6, you must manually create remediations for USBGuard-related rules.
++
+Additionally, remediations are created only for rules that satisfy minimum version requirements. link:https://bugzilla.redhat.com/show_bug.cgi?id=1965511[(BZ#1965511)]
```

Comment:
```suggestion
Additionally, remediations are created only for rules that satisfy minimum version requirements. [link:https://bugzilla.redhat.com/show_bug.cgi?id=1965511(*BZ#1965511*)]
```

Again, these formatting conventions are those used for the OCP release notes.

Reply:
Done (also fixed other references in the file for consistency)